pandas/core/reshape.py

```diff
@@ -61,7 +61,7 @@ class _Unstacker(object):
     """
 
     def __init__(self, values, index, level=-1, value_columns=None,
-                 fill_value=None):
+                 fill_value=None, frame_constr=DataFrame):
```

Comment:
Should be ``return_type`` if actually required, though I don't looked into it yet...